src/hotspot/share/c1/c1_CFGPrinter.hpp

```diff
@@ -62,7 +62,8 @@ class CFGPrinterOutput : public CHeapObj<mtCompiler> {
 
   void inc_indent();
   void dec_indent();
-  void print(const char* format, ...) ATTRIBUTE_PRINTF(2, 3);
+  ATTRIBUTE_PRINTF(2, 3)
+  void print(const char* format, ...);
```

Comment:
This is an example where rearranging the attributes is out of character with usual practice.
And I think it makes it harder to read.

Reply:
I agree with Kim, I do not like this style of using a new line for the attribute. I also prefer to see these attributes in their original location where I can generally ignore them while reading the code.